docs/configuration/http_conn_man/headers.rst

```diff
@@ -218,9 +218,9 @@ x-ot-span-context
 The *x-ot-span-context* HTTP header is used by Envoy to establish proper parent-child relationships
 between tracing spans. For the Lightstep tracer, *x-ot-span-context* is a base64 encoded
 `binary OT <https://github.com/opentracing/basictracer-go/blob/master/wire/wire.proto>`_
```

Comment:
pedantic nitpick: 
```
`binary "basictracer" < ... >` carrier
```

(OpenTracing per se does not have an "official" encoding here)

Reply:
This text was in the original version - I was actually thinking about removing it as the service developer does not need to know anything about the format.

I'll remove and if anyone has any objections, I can add some modified text back.